### PR TITLE
Orgs

### PR DESCRIPTION
**Pull request by m4s-b3n**
squash pls

### CODE REVIEW

File: .github/workflows/update-repo-request-template.yml [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+          yamllint_file_or_dir: ${{ env.ISSUE_TEMPLATE_FILE }}
+          yamllint_strict: true
+      - name: Commit Changes          
+        if: ${{ steps.change-detection.outputs.changed == 'true' }}
+        env: 
+          CI_COMMIT_MESSAGE: "chore(org): Update Issue Template File"
+          CI_COMMIT_AUTHOR: Continuous Integration
+        run: |
+          git config --global user.name "${{ env.CI_COMMIT_AUTHOR }}"
+          git config --global user.email "CI@users.noreply.github.com"
```
Comment (aatmmr) [inline .github/workflows/update-repo-request-template.yml:79]: Can this be achieved using a GitHub App? We have an app available in the xpirit-training repo that we can use for that.

File: .github/workflows/update-repo-request-template.yml [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+        with:
+          yamllint_file_or_dir: ${{ env.ISSUE_TEMPLATE_FILE }}
+          yamllint_strict: true
+      - name: Commit Changes          
+        if: ${{ steps.change-detection.outputs.changed == 'true' }}
+        env: 
+          CI_COMMIT_MESSAGE: "chore(org): Update Issue Template File"
+          CI_COMMIT_AUTHOR: Continuous Integration
+        run: |
+          git config --global user.name "${{ env.CI_COMMIT_AUTHOR }}"
```
Comment (spindev) [inline .github/workflows/update-repo-request-template.yml:78]:
```suggestion
          git config --global user.name "GitHub (Bot)"
```

File: .github/workflows/update-repo-request-template.yml (hunk 1 above)
Comment (spindev) [inline .github/workflows/update-repo-request-template.yml:79]:
```suggestion
          git config --global user.email "bot@github.com"
```

**Review by spindev — CHANGES_REQUESTED** [review]
User for the git commands changed to GitHub bot as suggestion.

File: README.md [hunk 3]
```diff
@@ -3,3 +3,39 @@
 Custom ISSUE_TEMPLATE for creating new repositories via issue forms.
 
 Workflow for auto provisioning new repositories based on issues and administer them.
+
+
+# queries
+
+## jq
+
+### create repo name
```
Comment (aatmmr) [inline README.md:12]:
```suggestion
# Queries

## jq

### Create repo name
```

File: README.md [hunk 4]
```diff
@@ -3,3 +3,39 @@
...
+PROJECT=$(jq -r '.Project' [issue-body] | sed 's/ /-/g' | tr '[:upper:]' '[:lower:]')
+
+RNAME=$(jq -r '.Name' [issue-body] | sed 's/ /-/g' | tr '[:upper:]' '[:lower:]')
+
+REPO=${COMPANY}-${TEAM}-${PROJECT}-${RNAME}
+```
+
+## yq
+
+### replace companies, teams & projects
```
Comment (aatmmr) [inline README.md:28]:
```suggestion
### Replace companies, teams & projects
```

File: .gitignore [hunk 5]
```diff
@@ -0,0 +1,6 @@
+# dev
+/companies
+/teams
+/projects
+/tmp
+/tmp_out
```
Comment (aatmmr) [inline .gitignore:6]: Is that local dev output?

**Review by aatmmr — CHANGES_REQUESTED** [review]
Awesome 🥳 🎉

File: .gitignore (hunk 5 above)
Comment (m4s-b3n) [reply .gitignore:6]: yes

Comment (m4s-b3n) [reply .gitignore:6]: used to add files with '-A' flag and this prevents from checking in temp files when running the commands from README.md

**Review by spindev — APPROVED** [review]
LGTM

**Pull request merged**